### PR TITLE
Upgradetest

### PR DESCRIPTION
**Pull request by jzoon**
"As a developer
I want to test all of my code
so that I'm sure that it works properly"

### CODE REVIEW

File: TU-Agent/upgrade.test2g [hunk 1]
```diff
@@ -0,0 +1,33 @@
+use TUAgent as mas.
+use agentKnowledge as knowledge.
+use upgrade as module.
+use tygron as actionspec.
+
+timeout = 30.
+
+% This test will test the buildConstruction module.
+test upgrade with
+	% Check if there is indeed the goal to build high buildings.
```
Comment (tvleest) [inline TU-Agent/upgrade.test2g]: Please change this comment.

File: TU-Agent/upgrade.test2g [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		never done(popup_answer(_,_)).
+		never done(popup_answer_with_date(_,_,_)).
+		
+		% If the module is started, something always needs to be upgraded.
+		bel(upgrades(UpgradeList)) leadsto
+			building_plan_upgrade(_,_).
+		
+		eventually not(bel(upgrades(UpgradeList))).
+	}
+	% We are not able to define a good post condition here since there can be multiple outcomes.
```
Comment (tvleest) [inline TU-Agent/upgrade.test2g]: Does this comment apply here?

File: TU-Agent/upgrade.test2g [hunk 3]
```diff
@@ -0,0 +1,33 @@
+use TUAgent as mas.
+use agentKnowledge as knowledge.
+use upgrade as module.
+use tygron as actionspec.
+
+timeout = 30.
+
+% This test will test the buildConstruction module.
```
Comment (tvleest) [inline TU-Agent/upgrade.test2g]: Please change this comment.

**Comment by MichielvdBerg** [comment]
If the comments are fixed, I approve.

**Comment by tvleest** [comment]
Better now with the correct comments.
Approved.

**Comment by MichielvdBerg** [comment]
Approved with the new comments.

**Pull request merged**